README.md

```diff
@@ -32,11 +31,6 @@ to collect feedback provided in the form of
 workflow [`step-summary`][step-summary], and Pull Request reviews (with
 [`tidy-review`][tidy-review] or [`format-review`][format-review]).
 
-> [!WARNING]
-> We only support Linux runners using a Debian-based Linux OS (like Ubuntu and many others).
->
-> MacOS and Windows runners are supported as well.
-
```

Comment:
Where did this warning go? The info is still accurate. At first, I thought you moved it down the page (which was fine), but now it is completely gone?!